packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -74,7 +108,38 @@ export function DatePicker({
                     </div>
                     <table data-testid="jkl-datepicker-calendar" />
                 </CoreDatepicker>
-            </CoreToggle>
+            </div>
+            <SupportLabel errorLabel={errorLabel} helpLabel={helpLabel} />
         </div>
     );
 }
+
+export function isSameDay(date1: Date, date2: Date) {
+    return (
+        date1.getDate() === date2.getDate() &&
+        date1.getMonth() === date2.getMonth() &&
+        date1.getFullYear() === date2.getFullYear()
+    );
+}
+
+/**
+ *
+ * @param date the date to format
+ * @return returns a date with "dd.mm.yyyy"-format
+ */
+export function formatDate(date: Date) {
+    const day = pad0(date.getDate());
+    const month = pad0(date.getMonth() + 1);
+    return `${day}.${month}.${date.getFullYear()}`;
+}
+
+/**
+ * @return returns a string with 0 in front of single digit numbers
+ */
+function pad0(nr: number) {
```

Comment:
Also, I might be in the minority here. Its pretty to define the function after usage, but I strongly dislike it. To me it feels like abuse of language design flaw. I.E someone wants to refactor into an arrow functions, and it just stops working.